src/routes/Dashboard/Dashboard.svelte

```diff
@@ -432,12 +466,66 @@
     padding: 0;
     margin: 0;
     flex-direction: column;
+    justify-content: space-around;
+    grid-area: all-measurements;
   }
-  @media (max-width: 700px) {
-    .all-measurements {
+
+  .all-charts {
+    display: grid;
+    grid-template-areas:
+      "chart1 chart2"
+      "chart3 chart4"
+      "chart5 chart5";
+    column-gap: 0.5rem;
+  }
+
+  .chart1 {
+    grid-area: chart1;
+    max-width: 475px;
+  }
+
+  .chart2 {
+    grid-area: chart2;
+    max-width: 475px;
+  }
+
+  .chart3 {
+    grid-area: chart3;
+    max-width: 475px;
+  }
+
+  .chart4 {
+    grid-area: chart4;
+    max-width: 475px;
```

Comment:
Super trivial, but might be worth consolidating these `475px` properties into one rule.

Reply:
done